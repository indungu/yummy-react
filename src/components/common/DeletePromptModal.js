import React from 'react';

export default (props) => {
  return (
    <div className="modal fade show" tabIndex="-1" role="dialog" id={props.toBeDeleted} style={{ display: props.display }}>
      <div className="modal-dialog" role="document">
        <div className="modal-content">
          <div className="modal-header">
            <h5 className="modal-title">Are you sure you want to delete this {props.query}?</h5>
            <button type="button" className="close" onClick={props.hide}>
              <span aria-hidden="true">&times;</span>
            </button>
          </div>
          <div className="modal-body">
            <p>{props.toBeDeleted}</p>
          </div>
          <div className="modal-footer">
            <button type="button" className="btn btn-secondary" onClick={props.hide}>Close</button>
            <button type="button" className="btn btn-danger" onClick={props.handleDelete}>Delete</button>
          </div>
        </div>
      </div>
    </div>
  );
};
