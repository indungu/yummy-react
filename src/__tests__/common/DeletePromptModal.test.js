import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import DeletePromptModal from '../../components/common/DeletePromptModal';

describe('Login', () => {
  const wrapper = shallow(<DeletePromptModal />);
  it('Renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
