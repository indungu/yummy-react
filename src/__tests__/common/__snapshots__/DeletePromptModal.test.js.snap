// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Renders properly 1`] = `
<div
  className="modal fade show"
  role="dialog"
  style={
    Object {
      "display": undefined,
    }
  }
  tabIndex="-1"
>
  <div
    className="modal-dialog"
    role="document"
  >
    <div
      className="modal-content"
    >
      <div
        className="modal-header"
      >
        <h5
          className="modal-title"
        >
          Are you sure you want to delete this 
          ?
        </h5>
        <button
          className="close"
          type="button"
        >
          <span
            aria-hidden="true"
          >
            Ã—
          </span>
        </button>
      </div>
      <div
        className="modal-body"
      >
        <p />
      </div>
      <div
        className="modal-footer"
      >
        <button
          className="btn btn-secondary"
          type="button"
        >
          Close
        </button>
        <button
          className="btn btn-danger"
          type="button"
        >
          Delete
        </button>
      </div>
    </div>
  </div>
</div>
`;
